<html><body><pre>
yarn run v1.22.19
$ eslint .

/home/jenkins/workspace/g_ui-plugin-find-eresource_PR-66/project/src/Container.js
  8:53  error  Unable to resolve path to module '@k-int/stripes-kint-components'  import/no-unresolved

✖ 1 problem (1 error, 0 warnings)

info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
</pre><body></html>

--- a/package.json
+++ b/package.json
@@ -61,6 +61,7 @@
   },
   "dependencies": {
     "@folio/stripes-erm-components": "^6.2.0",
+    "@k-int/stripes-kint-components": "^2.7.0",
     "@folio/stripes-testing": "^4.2.0",
     "@testing-library/react": "^12.1.2",
     "dom-helpers": "^3.4.0",
@@ -83,7 +84,8 @@
       "erm": "1.0 2.0 3.0 4.0 5.0"
     },
     "stripesDeps": [
-      "@folio/stripes-erm-components"
+      "@folio/stripes-erm-components",
+      "@k-int/stripes-kint-components"
     ],
     "permissionSets": [
       {

--- a/src/Container.js
+++ b/src/Container.js
@@ -3,9 +3,10 @@ import PropTypes from 'prop-types';
 
 import { useQuery } from 'react-query';
 
+import { generateKiwtQueryParams, useRefdata } from '@k-int/stripes-kint-components';
+
 import { useOkapiKy } from '@folio/stripes/core';
 import { getRefdataValuesByDesc, useInfiniteFetch } from '@folio/stripes-erm-components';
-import { generateKiwtQueryParams, useRefdata } from '@k-int/stripes-kint-components';
 
 import View from './View';
 

--- a/src/Container.js
+++ b/src/Container.js
@@ -55,7 +55,7 @@ const Container = ({
   // We only need local session query here, use state
   const [query, setQuery] = useState({});
   const querySetter = ({ nsValues }) => {
-    setQuery(nsValues);
+    setQuery({ ...query, ...nsValues });
   };
   const queryGetter = () => query;
 
@@ -92,7 +92,7 @@ const Container = ({
     results: eresources = [],
     total: eresourcesCount = 0
   } = useInfiniteFetch(
-    [ERESOURCES_ELECTRONIC_ENDPOINT, eresourcesQueryParams, 'ui-agreements', 'EresourcesRoute', 'getEresources'],
+    ['ERM', 'EResources', eresourcesQueryParams, ERESOURCES_ELECTRONIC_ENDPOINT],
     ({ pageParam = 0 }) => {
       const params = [...eresourcesQueryParams, `offset=${pageParam}`];
       return ky.get(encodeURI(`${ERESOURCES_ELECTRONIC_ENDPOINT}?${params?.join('&')}`)).json();
@@ -101,7 +101,7 @@ const Container = ({
 
   const kbsPath = 'erm/kbs';
   const { data: kbs = [] } = useQuery(
-    [kbsPath, 'ui-agreements', 'EresourcesRoute', 'getKbs'],
+    ['ERM', 'KnowledgeBases', kbsPath],
     () => ky.get(kbsPath).json()
   );
 

--- a/src/View.js
+++ b/src/View.js
@@ -81,7 +81,6 @@ const EResources = ({
       <SearchAndSortQuery
         initialFilterState={initialFilterState}
         initialSearchState={{ query: '' }}
-        // FIXME WIP the initialSortState does not get set
         initialSortState={{ sort: 'name' }}
         queryGetter={queryGetter}
         querySetter={querySetter}

